components/formats-bsd/src/loci/formats/in/FakeReader.java

```diff
@@ -1083,9 +1083,9 @@ private int populateSPW(MetadataStore store, int screens, int plates, int rows,
     final XMLMockObjects xml = new XMLMockObjects();
     OME ome = null;
     if (screens==0) {
-      ome = xml.createPopulatedPlate(plates, rows, cols, fields, acqs);
+      ome = xml.createPopulatedPlate(plates, rows, cols, fields, acqs, false);
```

Comment:
Good point. It would certainly be good to have this feature preserved if anyone depended on it. Let me see how hard it is to fix the AnnotationRef indexation issue in this PR. Otherwise, I'll tackle it in a follow-up PR.